qoqo_for_braket/qoqo_for_braket/interface/ionq_verbatim_interface.py

```diff
@@ -64,8 +64,8 @@ def call_circuit(circuit: qoqo.Circuit) -> Circuit:
             braket_circuit.ms(
                 op.control(),
                 op.target(),
-                0.0 - qubit_phase.get(op.control(), 0.0),
-                0.0 - qubit_phase.get(op.target(), 0.0),
+                0.0 + qubit_phase.get(op.control(), 0.0),
+                0.0 + qubit_phase.get(op.target(), 0.0),
```

Comment:
Is there a unittest that would have caught this bug?

Reply:
I have added the relevant unittest